doc/DeveloperGuide.md

```diff
@@ -87,6 +87,39 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+### Use case: renaming an existing tag
+
+**MSS**
+
+1. User requests to rename an existing tag
+2. AddressBook shows a list of existing tags
+3. User requests to rename a specific tag in the list to a new tag name
+4. AddressBook confirms the change with user
+5. User confirms the change
```

Comment:
Keep in mind that (especially for the project) that the app should have `advanced CLI`.